samples/monitor/create_scheduled_query_rule.py

```diff
@@ -0,0 +1,139 @@
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+import os
+
+from azure.common.credentials import ServicePrincipalCredentials
+from azure.identity import EnvironmentCredential
+from azure.mgmt.monitor import MonitorClient
+from azure.mgmt.loganalytics import LogAnalyticsManagementClient
+from azure.mgmt.resource import ResourceManagementClient
+
+
+# Cerdential for track1 sdk
+credential4track1 = ServicePrincipalCredentials(
```

Comment:
why do we have creds for track 1 ?

Reply:
Because some depended resources don't have track2 sdk now.